development/language/validation.rst

```diff
@@ -8,22 +8,22 @@ Submission guidelines
 Description
 -----------
 
-Contribution description in the `Customisation Database`_
+Contribution description in the `Customisations Database`_
```

Comment:
We use "Customisation Database" on phpBB.com e.g. in the navigation. So we have to change this everywhere or keep it here with "singular", too.